app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -81,6 +69,32 @@ def authorize_group_access
     raise NotAuthorized, "You are not permitted to select this team"
   end
 
+  def check_group_not_previous_acceptee
+    return unless group.present? && group_assignment_repo.present?
+
+    redirect_to successful_invitation_group_assignment_invitation_path
+  end
+
+  def check_user_not_group_member
+    return if group.blank?
+    redirect_to accept_group_assignment_invitation_path
```

Comment:
This method isn't new, it was just moved to a different spot in the file for organization purposes.